extensions/true-fantom/network/network-v001.js

```diff
@@ -0,0 +1,477 @@
+            text: 'redirect this tab to [URL]',
+            arguments: {
+              URL: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: 'https://extensions.turbowarp.org'
+              }
+            }
+          }
+        ],
+        
+        menus: {
+          content_type: { 
+            acceptReporters: true,
+            items: [
+              {
+                text: '(1) text',
+                value: '1'
+              },
+              {
+                text: '(2) json',
+                value: '2'
+              }
+            ]
+          },
+          only_url_response_type: {  
+            acceptReporters: true,
+            items: [
+              {
+                text: '(1) text',
+                value: '1'
+              },
+              {
+                text: '(2) json',
+                value: '2'
+              },
+              {
+                text: '(3) status ok?',
+                value: '3'
+              },
+              {
+                text: '(4) status',
+                value: '4'
+              },
+              {
+                text: '(5) status text',
+                value: '5'
+              },
+              {
+                text: '(6) type',
+                value: '6'
+              },
+              {
+                text: '(7) redirected?',
+                value: '7'
+              },
+              {
+                text: '(8) url',
+                value: '8'
+              }
+            ]
+          },
+          response_type: {  
+            acceptReporters: true,
+            items: [
+              {
+                text: '(1) text',
+                value: '1'
+              },
+              {
+                text: '(2) json',
+                value: '2'
+              },
+              {
+                text: '(3) status ok?',
+                value: '3'
+              },
+              {
+                text: '(4) status',
+                value: '4'
+              },
+              {
+                text: '(5) status text',
+                value: '5'
+              },
+              {
+                text: '(6) type',
+                value: '6'
+              },
+              {
+                text: '(7) redirected?',
+                value: '7'
+              },
+              {
+                text: '(8) url',
+                value: '8'
+              },
+              {
+                text: '(9) body used?',
+                value: '9'
+              }
+            ]
+          },
+          default: { 
+            acceptReporters: true,
+            items: [
+              {
+                text: 'default',
+                value: 'default'
+              }
+            ]
+          }
+        }
+      }
+    }
+    
+    connected_to_internet_block() {
+      return navigator.onLine;
+    }
+    browser_block() {
+      let has = input => navigator.userAgent.includes(input);
```

Comment:
Did you write this yourself or copy it from somewhere else? We should credit where we copy things from if that's the case

Reply:
Most of the changes from the update commit are from me, including this one

Reply:
I copied and modified this code `https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator`
(in version before @softedco 's changes)